File: sample/src/main/java/com/schibsted/spain/barista/sample/LabelActivity.java
package com.schibsted.spain.barista.sample;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.TextView;

public class LabelActivity extends AppCompatActivity {

  public static final String EXTRA_TEXT = "text";

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_centered_label);

    TextView selectedItem = (TextView) findViewById(R.id.selected_item);
    String value = getIntent().getStringExtra(EXTRA_TEXT);
    selectedItem.setText(value);
  }
}

File: sample/src/main/java/com/schibsted/spain/barista/sample/LabelActivity.kt
package com.schibsted.spain.barista.sample

import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class LabelActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_centered_label)
    val selectedItem = findViewById<View>(R.id.selected_item) as TextView
    val value = intent.getStringExtra(EXTRA_TEXT)
    selectedItem.text = value
  }

  companion object {
    const val EXTRA_TEXT = "text"
  }
}